langkit/__init__.py

```diff
@@ -35,6 +36,12 @@ class LangKitConfig:
         ]
     )
     reference_corpus: str = ""
+    injections_model_path: str = "JasperLS/gelectra-base-injection"
+    rouge_type: str = "rouge1"
+    sentiment_lexicon: str = "vader_lexicon"
+    topic_model_path: str = "MoritzLaurer/mDeBERTa-v3-base-xnli-multilingual-nli-2mil7"
+    topic_classifier: str = "zero-shot-classification"
+    toxicity_model_path: str = "martin-ha/toxic-comment-model"
```

Comment:
Not specific to your changes, but we should follow up on what the requirements are for swapping out some of these models, and have some examples.